tests-perf/locust/locust-notifications.py

```diff
@@ -1,24 +1,81 @@
-import os
+import sys, os
+sys.path.append(os.path.abspath(os.path.join("..", "tests_smoke")))
 
+from datetime import datetime
 from dotenv import load_dotenv
 from locust import HttpUser, constant_pacing, task
+from tests_smoke.smoke.common import job_line, rows_to_csv
 
 load_dotenv()
-AUTH_HEADER = os.getenv("TEST_AUTH_HEADER")
 
 
 class NotifyApiUser(HttpUser):
 
     wait_time = constant_pacing(60)
-    host = "https://api.staging.notification.cdssandbox.xyz"
+    host = os.getenv("LOAD_TEST_DOMAIN", "https://api.staging.notification.cdssandbox.xyz")
 
-    @task
-    def send_notifications(self):
-        headers = {"Authorization": AUTH_HEADER}
+    def __init__(self, *args, **kwargs):
+        super(NotifyApiUser, self).__init__(*args, **kwargs)
+
+        self.headers = {"Authorization": os.getenv("TEST_AUTH_HEADER")}
+        self.email = "success@simulator.amazonses.com"
+        self.template_id = "5ebee3b7-63c0-4052-a8cb-387b818df627"
```

Comment:
might be a good idea to move these template id's to env vars

Reply:
Yeah that could work.. It would require having them defined as template groups.. Example: EMAIL_TEMPLATE, EMAIL_WITH_ATTACHMENT_TEMPLATE, SMS_TEMPLATE etc etc??

Reply:
yeah, maybe prefaced by `LOAD_TEST_` or something, ie `LOAD_TEST_EMAIL_TEMPLATE`, etc.